Automattic/woocommerce-payments
# Minor tweak - remove space from i18n string

Related to #5897

#### Changes proposed in this Pull Request

In the  PR #5897 , I mistakenly [added a trailing space](https://github.com/Automattic/woocommerce-payments/pull/5897/files#diff-a41f2e2fca1befa933534f9c72b4d1552d19af998a9dc2252f592d31e10b7a59R382) within a translation tagged string. This is not a good practice, since translators should not be required to add spaces in strings. 

I have added the required space separately outside the translation string in this minor PR. 


#### Testing instructions

Existing tests should pass. 

For visual check - follow similar testing instructions as #5897 and make sure there is a space before the time remaining, in the notification. 


-------------------


**Post merge**
